### ISSUE TITLE
problem with <thread>

### ISSUE THREAD

**Issue by AntoninSonka** [MrZahourek/Cyberescape#3]
g++ main.cpp
In file included from main.cpp:18:
music.h: In function 'void playMusic(const CHAR*)':
music.h:34:14: error: 'std::this_thread' has not been declared
   34 |         std::this_thread::sleep_for(std::chrono::milliseconds(100));
      |              ^~~~~~~~~~~
main.cpp: In function 'int main()':
main.cpp:1737:13: error: 'thread' was not declared in this scope
 1737 |             thread musicThread(playMusic, filePath.c_str());
      |             ^~~~~~
main.cpp:19:1: note: 'std::thread' is defined in header '<thread>'; did you forget to '#include <thread>'?
   18 | #include "music.h"
  +++ |+#include <thread>
   19 |
main.cpp:3228:13: error: 'musicThread' was not declared in this scope
 3228 |             musicThread.detach();
      |             ^~~~~~~~~~~
main.cpp:3289:17: error: 'thread' was not declared in this scope
 3289 |                 thread musicThread(playMusic, filePath.c_str());
      |                 ^~~~~~
main.cpp:3289:17: note: 'std::thread' is defined in header '<thread>'; did you forget to '#include <thread>'?
main.cpp:4446:17: error: 'musicThread' was not declared in this scope
 4446 |                 musicThread.detach();
      |                 ^~~~~~~~~~~
main.cpp:4486:17: error: 'thread' was not declared in this scope
 4486 |                 thread musicThread(playMusic, filePath.c_str());
      |                 ^~~~~~
main.cpp:4486:17: note: 'std::thread' is defined in header '<thread>'; did you forget to '#include <thread>'?
main.cpp:5695:17: error: 'musicThread' was not declared in this scope
 5695 |                 musicThread.detach();
      |                 ^~~~~~~~~~~
In file included from renders.h:22,
                 from main.cpp:12:
myFce.h: In function 'int prepareShopTitle()':
myFce.h:693:1: warning: control reaches end of non-void function [-Wreturn-type]
  693 | }
      | ^

**Comment by AntoninSonka** [comment]
SOLVED
